cookiecutter.json

```diff
@@ -11,5 +11,8 @@
     "copyright_holder": "{{ cookiecutter.author_name }}",
     "transifex_project": "{{ cookiecutter.project_shortname }}",
     "database": ["postgresql", "mysql"],
-    "elasticsearch": ["6", "5"]
+    "elasticsearch": ["6", "5"],
+    "data_model": ["Custom (as part of the instance)", "No data model"],
```

Comment:
Hm, probably name them ``Custom`` and ``None`` and we'll have to document what they mean.